## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 443695`
Commit: [`1ae882f49c70edd258b3af725d6247d592a106e4`](https://github.com/NixOS/nixpkgs/commit/1ae882f49c70edd258b3af725d6247d592a106e4) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/1ae882f49c70edd258b3af725d6247d592a106e4..pull/443695/head))
Merge: [`320a7307a8f423b608967521ea3464ccb18b5f6a`](https://github.com/NixOS/nixpkgs/commit/320a7307a8f423b608967521ea3464ccb18b5f6a)

Logs: https://github.com/iedame/nixpkgs-review-gha/actions/runs/17794139529


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>backrest</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>backrest</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>backrest</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>backrest</summary>
<pre>Building subPackage ./internal/orchestrator/logging
Building subPackage ./internal/orchestrator/repo
Building subPackage ./internal/orchestrator/tasks
Building subPackage ./internal/protoutil
Building subPackage ./internal/queue
Building subPackage ./internal/resticinstaller
Building subPackage ./internal/testutil
Building subPackage ./pkg/restic
Building subPackage ./test/helpers
Building subPackage ./webui
buildPhase completed in 3 minutes 51 seconds
Running phase: checkPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;checkPhase&quot; }
panic: open /etc/protocols: operation not permitted

goroutine 1 [running]:
modernc.org/libc/honnef.co/go/netdb.init.0()
	/nix/build/nix-build-backrest-1.9.2.drv-9079-3653977858/source/vendor/modernc.org/libc/honnef.co/go/netdb/netdb.go:700 +0x713
FAIL	github.com/garethgeorge/backrest/internal/api	1.324s
FAIL</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>backrest</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>backrest</summary>
<pre>Building subPackage ./internal/orchestrator/logging
Building subPackage ./internal/orchestrator/repo
Building subPackage ./internal/orchestrator/tasks
Building subPackage ./internal/protoutil
Building subPackage ./internal/queue
Building subPackage ./internal/resticinstaller
Building subPackage ./internal/testutil
Building subPackage ./pkg/restic
Building subPackage ./test/helpers
Building subPackage ./webui
buildPhase completed in 53 seconds
Running phase: checkPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;checkPhase&quot; }
panic: open /etc/protocols: operation not permitted

goroutine 1 [running]:
modernc.org/libc/honnef.co/go/netdb.init.0()
	/nix/build/nix-build-backrest-1.9.2.drv-13865-3702401323/source/vendor/modernc.org/libc/honnef.co/go/netdb/netdb.go:700 +0x5b8
FAIL	github.com/garethgeorge/backrest/internal/api	0.025s
FAIL</pre>
</details>
</details>

--- a/pkgs/by-name/ba/backrest/package.nix
+++ b/pkgs/by-name/ba/backrest/package.nix
@@ -12,13 +12,13 @@
 }:
 let
   pname = "backrest";
-  version = "1.8.1";
+  version = "1.9.2";
 
   src = fetchFromGitHub {
     owner = "garethgeorge";
     repo = "backrest";
     tag = "v${version}";
-    hash = "sha256-lpYny+5bXIxj+ZFhbSn200sBrDShISESZw+L5sy+X+Q=";
+    hash = "sha256-3lAWViC9K34R8la/z57kjGJmMmletGd8pJ1dDt+BeKQ=";
   };
 
   frontend = stdenv.mkDerivation (finalAttrs: {
@@ -34,7 +34,7 @@ let
     pnpmDeps = pnpm_9.fetchDeps {
       inherit (finalAttrs) pname version src;
       fetcherVersion = 1;
-      hash = "sha256-q7VMQb/FRT953yT2cyGMxUPp8p8XkA9mvqGI7S7Eifg=";
+      hash = "sha256-vJgsU0OXyAKjUJsPOyIY8o3zfNW1BUZ5IL814wmJr3o=";
     };
 
     buildPhase = ''
@@ -62,7 +62,7 @@ buildGoModule {
       internal/resticinstaller/resticinstaller.go
   '';
 
-  vendorHash = "sha256-AINnBkP+e9C/f/C3t6NK+6PYSVB4NON0C71S6SwUXbE=";
+  vendorHash = "sha256-oycV8JAJQF/PNc7mmYGzkZbpG8pMwxThmuys9e0+hcc=";
 
   nativeBuildInputs = [
     gzip
@@ -97,6 +97,10 @@ buildGoModule {
     export BACKREST_RESTIC_COMMAND="${restic}/bin/restic"
     export HOME=$(pwd)
   '';
+
+  # skip tests on darwin due to /etc/protocols failure
+  # `__darwinAllowLocalNetworking = true;` wasn't sufficient
+  doCheck = !stdenv.isDarwin;
 
   postInstall = ''
     wrapProgram $out/bin/backrest \

